system_tests_tcp_adaptor_tls timing out after 400 seconds

```
 >       time.sleep(min(delay, remaining))
E       Failed: Timeout >400.0s

/home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_test.py:156: Failed
----------------- generated xml file: /__w/skupper-router/skupper-router/skupper-router/build/tests/junitxmls/system_tests_tcp_adaptor_tls.xml -----------------
=================================================================== short test summary info ====================================================================
ERROR ::TcpTlsAdaptor::test_01_tcp_basic_connectivity - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_100_memory_metrics - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_10_tcp_INTA_INTA_100 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_11_tcp_INTA_INTA_1000 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_12_tcp_INTA_INTA_500000 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_13_tcp_EA1_EC2_500000 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_14_tcp_EA2_EC1_2000000 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_20_tcp_connect_disconnect - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_50_concurrent_edges - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_50_concurrent_interior - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_70_half_closed_INTA_INTA - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_71_half_closed_INTA_INTB - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_72_half_closed_INTA_INTC - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_73_half_closed_EA1_EA1 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_74_half_closed_EA1_EB1 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_75_half_closed_EA1_EC2 - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_76_half_closed_EA1_EC2_large_message - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_77_half_closed_INTA_INTC_large_message - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_80_balancing - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_90_stats - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_authenticate_peer - Failed: Timeout >400.0s
ERROR ::TcpTlsAdaptor::test_authenticate_peer_fail - Failed: Timeout >400.0s
```

https://github.com/skupperproject/skupper-router/actions/runs/6644869850/job/18054947042#step:34:18779

--- a/tests/system_tests_tcp_adaptor.py
+++ b/tests/system_tests_tcp_adaptor.py
@@ -453,7 +453,7 @@ class TcpAdaptorBase(TestCase):
                                           'encapsulation': encapsulation,
                                           'siteId': cls.site}
             if mode == "interior":
-                router_dict = {'mode': mode, 'id': name, 'dataConnectionCount': '4'}
+                router_dict = {'mode': mode, 'id': name, 'dataConnectionCount': '4', "helloMaxAgeSeconds": '10'}
             else:
                 router_dict = {'mode': mode, 'id': name}
             config = [

